pkg/pillar/cmd/baseosmgr/handlebaseos.go

```diff
@@ -361,11 +386,21 @@ func doBaseOsInstall(ctx *baseOsMgrContext, uuidStr string,
 
 	for i, ctc := range config.ContentTreeConfigList {
 		cts := &status.ContentTreeStatusList[i]
-		if cts.RelativeURL != ctc.RelativeURL || !uuid.Equal(cts.ContentID, ctc.ContentID) {
+		// check that the contenttreeconfig and contenttreesrarys gace matching content ID
```

Comment:
Can't parse this comment

Reply:
Wow, well that is some funny typos. I have no idea how we got there. Fixing it now.